recipe/meta.yaml

```diff
@@ -12,23 +12,16 @@ source:
     - 0301-update-lexical-core-because-0.7.4-doesnot-compile.patch
 
 build:
-  number: 3
+  number: 1
 
 requirements:
   build:
     - {{ compiler('cxx') }}
     - {{ compiler('rust') }}
-    # Use pins to control cos6/cos7 match
-    - libgcc-ng  {{ libgcc }}
-    - libstdcxx-ng  {{ libstdcxx }}
   host:
     - pip {{ pip }}
     - python {{ python }}
-    - rust-nightly {{ rust_nightly }}
```

Comment:
I was thinking this would be needed, but as
```
   - rust {{ rust }}
```

Reply:
`rust_compiler` pulls `rust` in build env and build goes fine. I have added pin for `rust_compiler` in https://github.com/open-ce/open-ce/pull/507 and removed pin for `rust`.

Reply:
added pin for rust as per our discussion.